packages/browser-sdk/package.json

```diff
@@ -30,5 +30,8 @@
     "eslint": "8.53.0",
     "tsup": "^6.7.0",
     "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "@solana/web3.js": "^1.98.2"
```

Comment:
I don't think we want to add solana/web3 js as a dependency. If we have to I would opt in for @solana/kit which is tree shakeable. Maxim wanted us to stay as lean as possible to limit bundle size and @solana/web3.js is pretty big.